benchmark/pom.xml

```diff
@@ -147,7 +147,7 @@
          <groupId>org.apache.maven.plugins</groupId>
```

Comment:
There shouldn't be a need to configure the plugin here after these changes. It is inherited from the parent pom. Please check library module.